app/src/main/java/org/covidwatch/android/ui/exposures/ExposuresFragment.kt

```diff
@@ -17,23 +18,25 @@ class ExposuresFragment : BaseFragment<FragmentExposuresBinding>() {
     override fun bind(
         inflater: LayoutInflater,
         container: ViewGroup?
-    ): FragmentExposuresBinding = FragmentExposuresBinding.inflate(inflater, container, false)
+    ): FragmentExposuresBinding {
+        return FragmentExposuresBinding.inflate(inflater, container, false)
+    }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
         with(binding) {
+            lifecycleOwner = this@ExposuresFragment
```

Comment:
What does this do?

Reply:
This gives a lifecycle instance to the databinding code so it can observe `LiveData` objects.